### PR TITLE
[cypress][SIMS #1541] Adding Institution data-cy ids - I 

### PR DESCRIPTION
**Pull request by velang01**
Trying to get away from using the text components as identifier for HTML elements

### CODE REVIEW

File: sources/packages/web/src/views/institution/AppInstitution.vue [hunk 1]
```diff
@@ -57,7 +61,9 @@
             <template v-for="(item, index) in menuItems" :key="index">
               <v-list-item :value="index" @click="item.command">
                 <v-list-item-title>
-                  <span class="label-bold">{{ item.label }}</span>
+                  <span class="label-bold" data-cy="settingsButton1">{{
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/AppInstitution.vue:64]: Why the `settingsButton1` was used here?
There will be many items rendered by the `v-for`, right?

File: sources/packages/web/src/views/institution/InstitutionProfile.vue [hunk 2]
```diff
@@ -1,7 +1,11 @@
 <template>
   <full-page-container>
     <template #header>
-      <header-navigator title="Manage institution" subTitle="Manage Profile" />
+      <header-navigator
+        title="Manage institution"
+        data-cy="manageProfileLandingPage"
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/InstitutionProfile.vue:6]: I saw sometimes this type of `data-cy` added to the `header-navigator` and sometimes to the `full-page-container`. What is the idea here?

File: sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue [hunk 3]
```diff
@@ -4,21 +4,26 @@
       <header-navigator
         :title="locationName"
         subTitle="Confirmation Of Enrolment"
+        data-cy="confirmationOfEnrolment"
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue:7]: Are those not too generic (data-cy="confirmationOfEnrolment")?
Is it not better to have something more specific like `data-cy="confirmationOfEnrolmentHeader"`?
Maybe I am not following the idea. Can you please elaborate on that?

Comment (velang01) [reply sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue:7]: I thought of keeping it more specific, but wanted to keep it consistent with the value in the `API` value in the formIO

File: sources/packages/web/src/views/institution/InstitutionProfile.vue (hunk 2 above)
Comment (velang01) [reply sources/packages/web/src/views/institution/InstitutionProfile.vue:6]: The id's in the header navigator are specifically for the landing page - just to make sure that the user is now navigated to the correct page. The id's in the full page containers are to retrieve the elements where the individual cy-ids cannot be assigned for the dynamic elements

File: sources/packages/web/src/views/institution/AppInstitution.vue (hunk 1 above)
Comment (velang01) [reply sources/packages/web/src/views/institution/AppInstitution.vue:64]: I was not completely sure if there will only be one CTA at this point. But you are right - i ll probably change it to just settings button. Will update it to be more generic and handle using index in the tests

File: sources/packages/web/src/views/institution/InstitutionProfile.vue (hunk 2 above)
Comment (andrewsignori-aot) [reply sources/packages/web/src/views/institution/InstitutionProfile.vue:6]: I did not follow this part "cannot be assigned for the dynamic elements". Is it about the form.io forms?
For the header, I would still suggest some sort of naming pattern like `manageProfileHeader`.

Comment (ann-aot) [reply sources/packages/web/src/views/institution/InstitutionProfile.vue:6]: same qn

File: sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue (hunk 3 above)
Comment (velang01) [reply sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue]: updated it

File: sources/packages/web/src/views/institution/InstitutionProfile.vue (hunk 2 above)
Comment (velang01) [reply sources/packages/web/src/views/institution/InstitutionProfile.vue]: updated it

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the changes, looks good 👍 
About the possible workaround to add classes to the columns of the database, was it removed from this PR?

**Comment by velang01** [comment]
> Thanks for doing the changes, looks good 👍 About the possible workaround to add classes to the columns of the database, was it removed from this PR?

yeah i think i need some help from devs to do that.

**Review by ann-aot — APPROVED** [review]
👍

**Pull request merged**